apps/fxc-front/src/app/logic/score/scorer.ts

```diff
@@ -9,132 +7,24 @@ export enum CircuitType {
   OutAndReturn = 'Out and return',
 }
 
+export function getCircuitType(circuit?: OptimizerCircuitType) {
```

Comment:
Do we need to keep this function and the enum above ?

Reply:
I just tried to remove this enum but I had issues with importing the type from the lib. 
I have some issues on my laptop and I do not manage to launch the UI. As this modification affects UI code, I prefer to keep it as it for the moment.

Reply:
Ok. Removed this enum and the function. Keep the enum from @flyxc/optimizer